cli/bpmetadata/parser/state_parser.go

```diff
@@ -0,0 +1,49 @@
+package parser
+
+import (
+	"encoding/json"
+
+	"github.com/golang/protobuf/jsonpb"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+// ParseOutputTypesFromState parses the terraform.tfstate file and extracts the output types
+func ParseOutputTypesFromState(stateData []byte) (map[string]*structpb.Value, error) { // Change return type
```

Comment:
Lets add some tests for this with some sample statefiles (with outputs, complex output, no outputs)